lib/hexpm/repository/release.ex

```diff
@@ -189,12 +189,20 @@ defmodule Hexpm.Repository.Release do
   def latest_version(releases, opts) do
     only_stable? = Keyword.fetch!(opts, :only_stable)
     unstable_fallback? = Keyword.get(opts, :unstable_fallback, false)
+    with_docs? = Keyword.get(opts, :with_docs)
+
+    with_docs_releases =
+      if with_docs? do
+        Enum.filter(releases, & &1.has_docs)
+      else
+        releases
+      end
 
     stable_releases =
       if only_stable? do
-        Enum.filter(releases, &(to_version(&1).pre == []))
+        Enum.filter(with_docs_releases, &(to_version(&1).pre == []))
```

Comment:
Is it fine to filter twice to keep the code simpler or better to optimize this?

Reply:
Perfectly fine to filter twice 👍